Python/ceval.c

```diff
@@ -2069,6 +2069,32 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, PyFrameObject *f, int throwflag)
             DISPATCH();
         }
 
+        case TARGET(INT_ADD): {
+            PyObject *left = TOP();
+            if (PyLong_CheckExact(left)) {
+                PyObject *sum = _PyLong_AddInt((PyLongObject *)left, oparg);
+                Py_DECREF(left);
+                SET_TOP(sum);
+                if (sum == NULL) {
+                    goto error;
+                }
+                DISPATCH();
+            }
+            PyObject *right = PyLong_FromLongLong(oparg);
+            if (right == NULL) {
+                goto error;
```

Comment:
`Py_DECREF(left);` is not needed at this moment?

Reply:
No, `left` is still on the stack.